frontend/app/routes/$lang/_protected/alerts/index.tsx

```diff
@@ -17,17 +17,21 @@ export async function loader({ context: { session }, params, request }: LoaderFu
 
   const userInfoToken: UserinfoToken = session.get('userInfoToken');
 
-  invariant(userInfoToken.sin, 'Expected userInfoToken.sin to be defined');
+  invariant(userInfoToken.sub, 'Expected userInfoToken.sub to be defined');
 
   return redirect(getPathById('$lang/_protected/alerts/subscribe/index', params));
 
-  //TODO fix the redirection once we can retrieve the status of the subscription
+  //TODO: need to fix manager/index, subscribe/confirm, subscribe/index mock API
+  //const subscriptionService = getSubscriptionService();
+  //const user = await subscriptionService.getUserByRaoidcUserId(userInfoToken.sub);
 
-  /*if (!alertSubscription || alertSubscription.registered === false) {
-    return redirect(getPathById('$lang/_protected/alerts/subscribe/index', params));
-  }
-  if (alertSubscription.subscribed === true) {
+  /*if (user.emailVerified === true) {
     return redirect(getPathById('$lang/_protected/alerts/manage/index', params));
   }
-  return redirect(getPathById('$lang/_protected/alerts/subscribe/confirm', params));*/
+
+  if (user.email.length > 0) {
```

Comment:
```suggestion
  if (user.email !== undefined) {
```

Reason: the API should not return a blank string for an email address; it will either return undefined/null, or a valid email.

Reply:
fixed